site/hail.nginx.conf

```diff
@@ -32,8 +32,9 @@ server {
     }
 
     location ~ ^/docs/batch(|/.*)$ {
+        index index.html
         root /var/www/html;
-        try_files /docs/pipeline$1 /docs/batch$1 =404;
+        try_files /docs/pipeline$1 /docs/pipeline$1/ /docs/batch$1 /docs/batch$1/ =404;
```

Comment:
did you mean to duplicate /docs/batch$1?

Reply:
it's not duplicated, it appears again with a trailing `/`, without this line, it doesn't try the `index.html` page.